etcd/restore-etcd-cluster-on-coreos.md

```diff
@@ -0,0 +1,110 @@
+# Restore etcd Cluster on CoreOS
+
+Here you will find step-by-step instructions on how to recover your etcd cluster. It is important to know that you can not restore your etcd cluster using only discovery URL. Discovery URL is used only once at bootstrap.
+
+In this example we will review 3-members etcd cluster with one failed node. It is important to take into consideration that this example supposes you used [Cloud-Config][cloud-config] with [discovery URL][etcd-discovery] to bootstrap your cluster with the following default options:
+
+```yaml
+  etcd2:
+    advertise-client-urls: http://%PEER_IP_ADDRESS%:2379
+    initial-advertise-peer-urls: http://%PEER_IP_ADDRESS%:2380
+    listen-client-urls: http://0.0.0.0:2379,http://0.0.0.0:4001
+    listen-peer-urls: http://0.0.0.0:2380
+    discovery: %DISCOVERY_URL%
+```
+
+If you have etcd cluster with TLS, just use `https://` instead of `http://` in commands' examples below. Make sure you didn't set custom etcd [member name][etcd-member-name] (by default CoreOS uses [machine-id] for etcd member name). Let's assume that your etcd cluster has faulty `172.16.0.102` member:
```

Comment:
why do we care if they set a custom name for the member? if its important, we need to explain why.